app/src/main/res/values-de/strings.xml

```diff
@@ -44,29 +44,29 @@
     <string name="notification_settings_title">Benachrichtigung über eingehende Dateien anzeigen</string>
     <string name="overwrite_settings_title">Überschreiben erlauben</string>
     <string name="accept_settings_title">Transfer automatisch akzeptieren</string>
-    <string name="accept_setting_summary_on">Der Transfer wird automatisch akzeptiert</string>
-    <string name="accept_setting_summary_off">Jeder Transfer muss von Dir akzeptiert werden</string>
-    <string name="background_settings_title">Im Hintergrund ausführend</string>
+    <string name="accept_setting_summary_on">Der Transfer wird automatisch angenommen</string>
+    <string name="accept_setting_summary_off">Jeder Transfer muss von Dir bestätigt werden</string>
+    <string name="background_settings_title">Im Hintergrund ausführen</string>
     <string name="boot_settings_title">Automatisch starten</string>
     <string name="export_log_settings_tile">Debug-Protokoll in Datei schreiben</string>
     <string name="group_code_settings_title">Gruppenschlüssel</string>
     <string name="port_settings_title">Port</string>
     <string name="theme_settings_title">Thema</string>
-    <string name="notification_settings_summary">Dies wird möglicherweise durch Android\'s Benachrichtigungseinstellungen beeinflusst</string>
+    <string name="notification_settings_summary">Dies wird möglicherweise durch Androids Benachrichtigungseinstellungen beeinflusst</string>
```

Comment:
@uDEV2019 wrote, --> sphenotux comments
Line 55 must be "Android's". --> No, not in German, see below.
"Android" is an "own name" (Eigenname). --> correct
"Androids" looks awkward. --> May look akwward, but it is the correct use of the German "Genitiv". See
https://deutsch.lingolia.com/de/zeichensetzung/apostroph further "Genitiv mit Apostroph?" and further "Der Genitiv wird im Deutschen normalerweise ohne Apostroph gebildet; wir hängen das s direkt an das Nomen an." YES, this is different in English AFAIK.